htdocs/societe/class/api_thirdparties.class.php

```diff
@@ -1266,18 +1266,20 @@ public function updateCompanyBankAccount($id, $bankaccount_id, $request_data = n
 	public function deleteCompanyBankAccount($id, $bankaccount_id)
 	{
 		if (!DolibarrApiAccess::$user->hasRight('societe', 'creer')) {
-			throw new RestException(403);
+			throw new RestException(401);
```

Comment:
401 must be used when user failed to login.
403 when loggin is ok but permission are wrong.
So here, value should be 403.